modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -855,6 +855,8 @@ class CV_EXPORTS DeviceInfo
 CV_EXPORTS void printCudaDeviceInfo(int device);
 CV_EXPORTS void printShortCudaDeviceInfo(int device);
 
+CV_EXPORTS void convertFp16Cuda(InputArray _src, OutputArray _dst, Stream& stream = Stream::Null());
```

Comment:
I think `Cuda` prefix is redundant, since the function is already in `cuda::` namespace.